Scripts/update_xcodeproj_settings.rb

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/ruby
+
+require 'xcodeproj'
+
+# Extract and check the arguments
+# argv 1) The full path to the xcodeproj to be modified
+# argv 2) The name of the organisation to be applied to the project
+# argv 3) The name of the target to add the SwiftLint build phase to
+
+project_path = ARGV[0]
+
+if project_path == nil
+    puts "ERROR: Require file path to xcode project file in 1st argument"
+    exit
+elsif !project_path.end_with?(".xcodeproj")
+    puts "ERROR: Path doesn't include the correct file extension (.xcodeproj): #{project_path}"
+    exit
+elsif !File.exist?(project_path)
+    puts "ERROR: Project not found at supplied project path: #{project_path}"
+    exit
+end
+
+new_organisation_name = ARGV[1]
+
+if new_organisation_name == nil
+    puts "ERROR: Require organisation name in 2nd argument"
+    exit
+end
+
+target_name = ARGV[2]
+
+if target_name == nil
+    puts "ERROR: Require target name in 3nd argument"
```

Comment:
`3rd` not `3nd`

Reply:
ahah haven't wrote that, but it is literally everywhere